src/App.css

```diff
@@ -1,3 +1,64 @@
 .App {
-  text-align: center;
+  width: 100%;
+  height: 100vh;
+}
+
+:root {
+  --containersOpacity: 0;
+}
+
+#mapContainer {
+  width: 100%;
+  height: 100vh;
+  z-index: 0;
+}
+
+#upperLeftButtonsContainer {
+  width: 360px;
+  height: 120px;
+  background-color: rgba(64, 224, 208, var(--containersOpacity));
```

Comment:
why do we define colors to divs with opacity 0?

Reply:
See my reply on --containersOpacity

Reply:
I think we've talked about it. I think that it is a bad practice to add 'DEV' code to the master.
It's up to you 🥇

Reply:
yes, i agree and i have deleted it from the code, especially since it is very easy to re-add if i should need it.